<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnLibro</class>
 <widget class="QMainWindow" name="vtnLibro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>389</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sistema Invetario Libro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCodigo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Codigo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrecio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Precio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCantidad">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cantidad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAutor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Autor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEdicion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edici√≥n:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIsbn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIsbn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCantidad">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAutor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEdicion">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNuevo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>NUEVO</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLibrobuscar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::EditFind"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBorrar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BORRAR</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::EditDelete"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SyncSynchronizing"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPrecio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtCodigo</tabstop>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtPrecio</tabstop>
  <tabstop>txtCantidad</tabstop>
  <tabstop>txtIsbn</tabstop>
  <tabstop>txtAutor</tabstop>
  <tabstop>txtEdicion</tabstop>
  <tabstop>btnNuevo</tabstop>
  <tabstop>btnActualizar</tabstop>
  <tabstop>btnBorrar</tabstop>
  <tabstop>btnLimpiar</tabstop>
  <tabstop>btnLibrobuscar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
